api/output_policy.go

```diff
@@ -68,11 +66,11 @@ func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to unescape request URL characters: %v", err)
 	}
-	apiAddrPrefix := fmt.Sprintf("%sv1/", d.VaultClient.config.Address)
+	apiAddrPrefix := fmt.Sprintf("%sv1/", d.vaultServerAddress)
```

Comment:
I don't necessarily mind this logic, but I'm surprised that the bits you're interested in aren't available as part of `*retryablehttp.Request`

Reply:
I think you might be able to use `strings.TrimPrefix(d.URL.Path, "/v1")` and avoid passing in the server address altogether.